### DISCUSSION TITLE
2.9.1

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#544] [Releases]
# Upscayl v2.9.1 🥳

**ok, no bugs this time!**
<img width="484" alt="image" src="https://github.com/upscayl/upscayl/assets/25067102/f4d35c79-b511-4e11-9566-c5db3dd560f6">

## What's Changed 🤓
* **NCNN update!** Vulkan SDK and a lot of libraries have been updated, expect bug fixes and performance improvements💯 
* Fix MacOS App Store build. Launching soon! 📂
* Updated dependencies 🔗
* Fix #499 - Remove redundant upscaled files 🗑️
* Fix #488 - Batch Upscale empty progress bug 🪹
* Security updates 👮‍♀️
* Skip conversion for 4x PNG upscaling ⏭️

**Full Changelog**: https://github.com/upscayl/upscayl/compare/v2.8.6...v2.9.1

<hr /><em>This discussion was created from the release <a href='https://github.com/upscayl/upscayl/releases/tag/v2.9.1'>2.9.1</a>.</em>

**Comment by webdronez66** [comment]
Put that fly swatter away this time. well done

**Reply by NayamAmarshe** [reply]
No violence ☝️

**Comment by flottenheimer** [comment]
Just heard about and installed Upscayl a couple of hours ago. Wow! Great work you've done.

Can I find a general overview and comparison of the different models anywhere? Would be a timesaver to read about/see strengths/weaknesses rather than having to test and compare them all
.
Anyway, runs perfect on my M1 MacBook Pro. Looking forward to the App Store release. Keep up your great work.

**Reply by aaronliu0130** [reply]
I'm pretty sure I wrote about this somewhere already, but I can't find it and I've also forgotten...
In terms of power, I have the following entirely unfounded scale for upscayling power on general images:
Ultrasharp > Ultramix Balanced > Remacri > Fast Real-ESRGAN (which is only slightly worse in terms of quality than the normal Real-ESRGAN)
Keep in mind the higher the power, the more likely it is for a model to add unfounded details.
Obviously, Real-ESRGAN-animevideo (Digital Art) is for non-photographic/flat art. Note that "general images" still includes digital art, it just might recognize art as photos sometimes.

**Comment by JonMike12341234** [comment]
I have used upscale for months, and finally decided to upgrade.  Now its broke and won't do batch images (decoding error).  That's why i hate updates.  If its not broke don't fix it.  But I thought it would even be better.  Additionally, under the github releases they removed past versions.  Help!

**Reply by aaronliu0130** [reply]
Please reply in #584.

**Reply by moritztim** [reply]
>  That's why i hate updates. If its not broke don't fix it.

Shit take
